src/Storages/Kafka/StorageKafka.cpp

```diff
@@ -653,6 +710,19 @@ cppkafka::Configuration StorageKafka::getConsumerConfiguration(size_t consumer_n
     return conf;
 }
 
+cppkafka::Configuration StorageKafka::getProducerConfiguration()
+{
+    cppkafka::Configuration conf;
+    conf.set("metadata.broker.list", brokers);
+    conf.set("client.id", client_id);
+    conf.set("client.software.name", VERSION_NAME);
+    conf.set("client.software.version", VERSION_DESCRIBE);
```

Comment:
Cosmetic: + newline before l. 720, also in l. 702.

Reply:
Done.